MetricsDashboard.Core/Models/BooleanTile.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using MetricsDashboard.Contract;
+using MetricsDashboard.Contract.Enums;
+using MetricsDashboard.Core.Entities;
+
+namespace MetricsDashboard.Core.Models
+{
+    public class BooleanTile : ITile
+    {
+        private readonly string _name;
+
+        private readonly MetricEntity<bool> _latest;
```

Comment:
In this approach Mongo Entities shouldnt be part of Domain Model.

Reply:
removed Domain Models